tests/acceptance/test_dcos_zookeeper.py

```diff
@@ -24,3 +24,8 @@ def test_delete_zk_node():
     uninstall_package_and_wait('marathon')
 
     assert delete_zk_node('universe/marathon-user')
+
+
+def get_zk_node_data():
+    install_package_and_wait('marathon')
+    get_zk_node_data('universe/marathon-user')
```

Comment:
I think the last statement of a `test_` method will assume `assert` -- this could be more explicitly written as `assert get_zk_node_data('universe/marathon-user')` but I _think_ the same thing is being accomplished.